Gordon360/Services/HousingService.cs

```diff
@@ -0,0 +1,774 @@
+
+            DateTime now = System.DateTime.Now;
+
+            appIDParam = new SqlParameter("@APPLICATION_ID", applicationID);
+            SqlParameter timeParam = new SqlParameter("@NOW", now);
+            if (newEditorID != storedEditorID)
+            {
+                SqlParameter editorParam = new SqlParameter("@EDITOR_ID", userID);
+                SqlParameter newEditorParam = new SqlParameter("@NEW_EDITOR_ID", newEditorID);
+                int? result = context.Database.ExecuteSqlCommand("UPDATE_AA_APPLICATION_EDITOR @APPLICATION_ID, @EDITOR_ID, @NOW, @NEW_EDITOR_ID", appIDParam, editorParam, timeParam, newEditorParam); //run stored procedure
+                if (result == null)
+                {
+                    throw new ResourceCreationException() { ExceptionMessage = "The application could not be updated." };
+                }
+            }
+            else
+            {
+                int? result = context.Database.ExecuteSqlCommand("UPDATE_AA_APPLICATION_DATEMODIFIED @APPLICATION_ID, @NOW", appIDParam, timeParam); //run stored procedure
+                if (result == null)
+                {
+                    throw new ResourceCreationException() { ExceptionMessage = "The application DateModified could not be updated." };
+                }
+            }
+
+            return applicationID;
+        }
+
+        /// <summary>
+        /// Changes the student user who has permission to edit the given application
+        ///
+        /// </summary>
+        /// <returns>Whether or not all the queries succeeded</returns>
+        public bool ChangeApplicationEditor(string userID, int applicationID, string newEditorID)
+        {
+            IEnumerable<string> editorResult = null;
+
+            SqlParameter appIDParam = new SqlParameter("@APPLICATION_ID", applicationID);
+
+            editorResult = RawSqlQuery<string>.query("GET_AA_EDITOR_BY_APPID @APPLICATION_ID", appIDParam);
+            if (editorResult == null)
+            {
+                throw new ResourceNotFoundException() { ExceptionMessage = "The application could not be found." };
+            }
+            else if (!editorResult.Any())
+            {
+                return false;
+            }
+
+            string storedEditorID = editorResult.FirstOrDefault();
+
+            if (userID != storedEditorID)
+            {
+                // Return false if the current user does not match this application's editor stored in the database
+                return false;
+            }
+            // Only perform the update if the ID of the current user matched the 'EditorID' ID stored in the database for the requested application
+
+            CCTEntities1 context = new CCTEntities1();
+
+            DateTime now = System.DateTime.Now;
+
+            appIDParam = new SqlParameter("@APPLICATION_ID", applicationID);
+            SqlParameter timeParam = new SqlParameter("@NOW", now);
```

Comment:
Same as above, no need to save Now to a variable when you only reference it once.